docs/get_sql_db.md

```diff
@@ -0,0 +1,43 @@
+# get_sql_db
```

Comment:
For each of the "id" arguments (i.e instance_id, availability_group_id etc.) can we dynamically look them up instead of having the end user provide them?